test/test_jit.py

```diff
@@ -5043,7 +5043,59 @@ def test_math_ops(self):
         def test_floor():
             return math.floor(1.5)
 
+        def test_ceil():
+            return math.ceil(1.5)
+
+        def test_log_int():
+            return math.log(2)
+
+        def test_log_float():
+            return math.log(2.0)
+
+        def test_log1p_int():
+            return math.log1p(1)
+
+        def test_log1p_float():
+            return math.log1p(1.0)
+
+        def test_log10_int():
+            return math.log10(2)
+
+        def test_log10_float():
+            return math.log10(2.0)
+
+        def test_exp_int():
+            return math.exp(2)
+
+        def test_exp_float():
+            return math.exp(2.0)
+
+        def test_sqrt_int():
+            return math.sqrt(2)
+
+        def test_sqrt_float():
+            return math.sqrt(2.0)
+
+        def test_pow_float():
+            return math.pow(2.0, 2.0)
+
+        def test_pow_int():
+            return math.pow(2.0, 2)
```

Comment:
For all these tests could you refactor them to take in an input instead of using a literal? This will let us test different values more easily.